data-prepper-plugins/opensearch/README.md

```diff
@@ -155,6 +157,15 @@ If a single record turns out to be larger than the set bulk size, it will be sen
 
 - `trace_analytics_service_map`: No longer supported starting Data Prepper 2.0. Use `index_type` instead.
 
+### <a name="aws_configuration">AWS Configuration</a>
+
+The AWS configuration is the same for both SQS and S3.
+
+* `region` (Optional) : The AWS region to use for credentials. Defaults to [standard SDK behavior to determine the region](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html).
+* `sts_role_arn` (Optional) : The AWS STS role to assume for requests to SQS and S3. Defaults to null, which will use the [standard SDK behavior for credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html).
```

Comment:
This references "SQS and S3" - please correct for OpenSearch service.

Reply:
I will remove that statement.